packages/styles/src/components/header/index.scss

```diff
@@ -5,6 +5,28 @@
 @use 'mixins' as *;
 
 post-header {
+  --global-header-height: 72px;
+  --local-header-height: 112px;
+  --local-header-min-height: 56px;
+  --header-height: calc(var(--global-header-height) + var(--local-header-height));
+  --global-header-reduced-height: var(--global-header-height);
```

Comment:
Can you place these properties either in the `media.min(lg)` or the `media.max(lg)` mixins for better readability? Currently it is not clear that the `--global-header-height: 72px;` is only true on large screens whereas the `--global-header-reduced-height: var(--global-header-height);` is only true on small screens. 

I would only keep the `--header-height` outside of the mixins since its calculated.